trend-chef-splunk-security/nested/orchestrator-server-setup-trend.json

```diff
@@ -38,9 +38,9 @@
         "autoUpgradeMinorVersion": true,
         "settings": {
           "fileUris": [
-            "https://raw.githubusercontent.com/sysgain/cloudtry-p2p-public-artifacts/master/scripts/TrendMicro/p2porchAllinOneTrend.sh"
+            "https://raw.githubusercontent.com/sysgain/cloudtry-p2p-public-artifacts/master/scripts/TrendMicro/p2porchAllinOneTrend1.sh"
```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#samples-that-contain-extra-artifacts-custom-scripts-nested-templates-etc) for how to handle artifacts